src/plugins/data_source_management/public/components/popover_button/popover_button.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import React from 'react';
+import { EuiButtonEmpty } from '@elastic/eui';
+import { i18n } from '@osd/i18n';
+
+interface PopoverButtonProps {
+  className: string;
+  label?: string;
+  onClick: () => void;
+}
+
+export const PopoverButton: React.FC<PopoverButtonProps> = ({ className, label, onClick }) => {
+  return (
+    <>
+      <EuiButtonEmpty
+        className={`dataSourceComponentButtonTitle`}
```

Comment:
can we use the className passed down?

Reply:
Yeah. It is the unified CSS selector to control the button length, make sure it does not exceed 16 ch. So will not need to pass down since it is unified among all pickers,